Atomic/sign.py

```diff
@@ -0,0 +1,89 @@
+from . import util
+from . import Atomic
+import os
+import tempfile
+from .atomic import AtomicError
+from .client import ids_same, add_shaw
+
+
+class Sign(Atomic):
+    def sign(self):
+        # TODO
+        # If an alternate sign key is provided, should we verify that it is valid?
+        # Should we check to see if there are valid GPG signatures?
```

Comment:
Adding more signatures should never hurt. Adding a completely redundant one might merit a warning, sure.